integration/util/scenario/validator.js

```diff
@@ -218,12 +218,15 @@ class Validator {
     let newCliArgs = [];
     let ethRpcUrl = this.ethProxy ? this.ethProxy.serverUrl() : this.ctx.eth.web3Url;
     if (this.version.supports('full-cli-args')) {
-      newCliArgs = [
+      if (this.version.supports('generic-cli-args')) {
+        newCliArgs.push('--')
```

Comment:
really wish i could do this without needing this small breaking change :(

Reply:
Why do we need to support multiple cli versions? Just for upgrade tests?

Reply:
it is mostly unfortunate that we will need to tell any validators that are using the cli args to move these args to the end after a `--` this code change is to support the upgrade tests